pkg/kubelet/config/defaults.go

```diff
@@ -17,11 +17,12 @@ limitations under the License.
 package config
 
 const (
-	DefaultKubeletPodsDirName             = "pods"
-	DefaultKubeletVolumesDirName          = "volumes"
-	DefaultKubeletVolumeDevicesDirName    = "volumeDevices"
-	DefaultKubeletPluginsDirName          = "plugins"
-	DefaultKubeletContainersDirName       = "containers"
-	DefaultKubeletPluginContainersDirName = "plugin-containers"
-	DefaultKubeletPodResourcesDirName     = "pod-resources"
+	DefaultKubeletPodsDirName                = "pods"
+	DefaultKubeletVolumesDirName             = "volumes"
+	DefaultKubeletVolumeDevicesDirName       = "volumeDevices"
+	DefaultKubeletPluginsDirName             = "plugins"
+	DefaultKubeletPluginsRegistrationDirName = "plugin_registry"
```

Comment:
@saad-ali did you want this to be named `plugins_registry`?

Reply:
Yes!! Good catch. @RenaudWasTaken